src/Parser/Core/Modules/Items/Legion/AntorusTheBurningThrone/NorgannonsProwess.js

```diff
@@ -32,11 +33,15 @@ class NorgannonsProwess extends Analyzer {
   };
 
   intProc = 0;
+  intBuff = 0;
   pantheonProc = 0;
   damage = 0;
 
   on_initialized() {
     this.active = this.combatants.selected.hasTrinket(ITEMS.NORGANNONS_PROWESS.id);
+    if(this.active) {
+      this.intBuff = calculatePrimaryStat(940, 3257, this.combatants.selected.getItem(ITEMS.NORGANNONS_PROWESS.id).itemLevel);
```

Comment:
The int value should be 11483 at 940 ilvl. The value in the proc on wowhead is usually way too low, but the tooltip value on the trinket should be correct: http://www.wowhead.com/item=154177/norgannons-prowess